hail/src/main/scala/is/hail/utils/package.scala

```diff
@@ -674,6 +674,19 @@ package object utils extends Logging
       ab.result()
     }
   }
+
+  def dumpClassLoader(cl: ClassLoader) {
```

Comment:
This isn't getting called anywhere. Are you leaving it in as a debugging aid?

Reply:
Yes.  I've written this several times now and it is essential for debugging class loader issues, so I figure we might as well leave it in.